phalcon/assets/collection.zep

```diff
@@ -59,6 +59,18 @@ class Collection implements \Countable, \Iterator
 
 	protected _sourcePath { get };
 
+	/**
+	 * Version of resource
+	 * @var string
+	 */
+	protected _version { get, set };
+
+	/**
+	 * Should version be determined from file modification time
+	 * @var bool
+	 */
+	protected _autoVersion = false { set };
```

Comment:
I propose to not introduce anymore _prefixed variables. Lets use `autoVersion` instead of `_autoVersion`.